guides/admin-manual/admin_cache_management.rst

```diff
@@ -0,0 +1,65 @@
+.. meta::
+   :description: Viewing and clearing DNS caches in Micetro
+   :keywords: DNS, DNS servers, Cache Management, Clear Cache
+
+.. _admin-cache-management:
+
+Managing the DNS cache
+======================
+
+From the :guilabel:`Admin` page, you can view a DNS server's cache and, if necessary, clear it. You may need to clear a server's cache in the following cases:
+
+* **Domain IP address changes** — When the domain IP address changes, the DNS server may still have the old IP cached. Clearing the cache ensures that all queries are resolved with the updated IP address.
+* **Security concerns** — If DNS cache poisoning or other malicious activities are suspected, clearing the cache helps remove potentially harmful entries.
+* **Troubleshooting** — If stale or incorrect DNS entries are causing connectivity issues, clearing the cache helps to verify if this is the cause of the problem.
+* **Load balancing** — If DNS-based load balancing is being used, clearing the cache ensures the DNS server provides updated IP addresses for balanced traffic distribution.
+* **Service migrations** — Clearing the cache ensures that clients are directed to the correct new locations when migrating services to new servers.
+* **Configuration updates** — Clearing the cache after updating DNS server configurations or policies ensures that all queries reflect the latest settings.
+
+..note::
+    Options to clear a server cache are available for MS and BIND DNS servers.
+
+
+Viewing the cache
+-----------------
+
+You can view the cache of a selected DNS server to review its entries and determine whether or not it needs to be cleared.
+
+**To view a DNS server cache:**
+
+1. On the :guilabel:`Service Management` tab of the :guilabel:`Admin` page, select the desired server from the list.
+
+2. Hover your cursor over the entry and select the the **menu bar** that appears.
+
+3. From the dropdown, select :guilabel:`Cache management`.
+
+4. In the modal, enter a cache entry name in the search bar and select :guilabel:`Search`.
+
+   .. image:: ../../images/_admin-cache-management.png
+      :width: 80%
+
+The entries in the DNS server's cache will be displayed in the modal.
```

Comment:
Just a thought. I think that we have more often used the naming "dialog" instead of modal when talking about these smaller screens. Could you check?

Reply:
Good catch! Thank you!